versions.yaml

```diff
@@ -40,7 +40,15 @@ git:
   opa:
     url: https://github.com/open-policy-agent/opa
     reference: v0.58.0
+  kbs:
+    url: https://github.com/confidential-containers/kbs
+    reference: 6a9be1cd9d1522d6892c36cb80c415af1078b1e2
 oci:
   pause:
     registry: docker://registry.k8s.io/pause
     tag: 3.6
+
+image:
+  kbs:
+    registry: ghcr.io/confidential-containers/key-broker-service
+    tag: v0.8.2
```

Comment:
why do we introduce a new "image" hierarchy in versions yaml, shouldn't we be using the same revision as in the"git" hierarchy, so "image.kbs.tag" would be redundant? I think we can can hardcode kbs.registry (not the tag) in the github action workflow.

Reply:
1. I think we shouldn't hardcode anything as people might be interested in using local image and other repos.
2. Also keeping it in git hierarchy makes it confusing as it only contains github branch and repo related things. I think `oci` is also created as separate tree so its okay to use image as separate tree as well. We can later on add any image to this tree.

Reply:
Yeah, I agree with Kartik, under git it might seem like `git.kbs.image.registry` & `git.kbs.image.tag`.

So, either we need each dependency at root level, and it should have `git` and `image` that would be ideal too. But for now we can either create a new root level key `image` or use the existing `oci`. I am not sure what was the purpose of creating `oci` though.

Reply:
according to the linked issue, the requirement is a reference for a kbs container that can be used for CI. so I assume that one should go into the existing `oci` hierarchy (or what would the the difference between image and oci be?)

what's unclear to me: why do we have a `git.kbs` entry, i.e. what is it being used for? and why does it have a different reference than the CI image. maybe there's a good reason for that, it's just not stated in the issue or the description.

Reply:
So this git kbs entry will be used to clone appropriate compatible kbs version in CI for provisioning kbs. I will mention that as well in issue.

Reply:
so if the git.kbs entry is used for CI, what is the oci.kbs entry being used for?

Reply:
Keeping only one oci.kbs tag which will be used for both git reference and kbs image tag.